sympy/geometry/line.py

```diff
@@ -201,7 +202,7 @@ def angle_between(l1, l2):
             raise TypeError('Must pass only LinearEntity objects')
 
         v1, v2 = l1.direction, l2.direction
-        return acos(v1.dot(v2)/(abs(v1)*abs(v2)))
+        return acos(v1.dot(v2) / (abs(v1) * abs(v2)))
```

Comment:
These changes can be removed. We use space around `+-` but not `*/` but this is loosely enforced. If you do 
```
git reset head~22 # or however many commits you have when you get to this
git add -p # then say y/n to each change as prompted, saying n to all these whitespace changes
git commit -m # add bisection method for Line' (or whatever title you want
git reset --hard # to throw away other changes
git push -f Jaime02 master
```

that should update things here under a single commit.